templates/expo-template-default/hooks/useColorScheme.web.ts

```diff
@@ -1,8 +1,27 @@
-// NOTE: The default React Native styling doesn't support server rendering.
-// Server rendered styles should not change between the first render of the HTML
-// and the first render on the client. Typically, web developers will use CSS media queries
-// to render different styles on the client and server, these aren't directly supported in React Native
-// but can be achieved using a styling library like Nativewind.
+import { useColorScheme as useRNColorScheme } from 'react-native'
+import { useState, useEffect } from 'react'
+
 export function useColorScheme() {
-  return 'light';
+  // NOTE: The default React Native styling doesn't support server rendering.
+  // Server rendered styles should not change between the first render of the HTML
```

Comment:
This comment isn't quite right. React Native Web does support SSR, the issue is that native doesn't support media queries. The Meta team is looking at adding media queries via StyleX which we can pull in later when it gets more traction.